Models/AnimatedColorsCube/README.body.md

```diff
@@ -0,0 +1,11 @@
+## Screenshot
+
+![screenshot](screenshot/screenshot-large.gif)
+
+## Description
+
+This model tests the [`KHR_animation_pointer`](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_animation_pointer) extension's effect on the `baseColorFactor` of a material.  When animated, the top cube should rotate above other colored cubes, and change its own color to match the cubes below.
```

Comment:
Please add more asset details.

Specifically, the animation features two time inputs: movement is defined for the `[0, 3]` range while colors are defined for the `[0, 2.5]` range. Assuming that it's intentional, this setup additionally checks that the last color keyframe remains active until the movement track is restarted.

Reply:
README updated.